README.md

```diff
@@ -3,28 +3,42 @@
 
-# global-workflow
-Global Workflow currently supporting the Global Forecast System (GFS) with the [UFS-weather-model](https://github.com/ufs-community/ufs-weather-model) and [GSI](https://github.com/NOAA-EMC/GSI)-based Data Assimilation System.
+The Global Workflow supporting the Global Forecast System (GFS), the Global Ensemble Forecasting System (GEFS), and the Seasonal Forecast System (SFS) with the [UFS-weather-model](https://github.com/ufs-community/ufs-weather-model).  Data assimilation, currently only available for the GFS, is provides by both the [GSI](https://github.com/NOAA-EMC/GSI)- and [GDASApp (JEDI)](https://github.com/NOAA-EMC/GDASApp)-based Data Assimilation systems.
 
-The `global-workflow` depends on the following prerequisities to be available on the system:
+In progress [documentation](https://global-workflow.readthedocs.io/en/latest/) is available.
 
-* Workflow Engine - [Rocoto](https://github.com/christopherwharrop/rocoto) and [ecFlow](https://github.com/ecmwf/ecflow) (for NWS Operations)
-* Compiler - Intel Compiler Suite
-* Software - NCEPLIBS (various), ESMF, HDF5, NetCDF, and a host of other softwares (see module files under /modulefiles for additional details)
+# Prerequisites
 
-The `global-workflow` current supports the following tier-1 machines:
+The Global Workflow depends on the following prerequisities to be available on the system:
 
-* NOAA RDHPCS - Hera
-* MSU HPC - Orion
-* MSU HPC - Hercules
-* NOAA's operational HPC - WCOSS2
+* Workflow Engines - [Rocoto](https://github.com/christopherwharrop/rocoto) and [ecFlow](https://github.com/ecmwf/ecflow) (for NWS Operations)
+* Compiler - Intel classic compiler suite version 2021.x
+* Software - NCEPLIBS (various), ESMF, HDF5, NetCDF, and a host of other softwares (see module files under /modulefiles for additional details).
+    - [Spack-stack](https://github.com/JCSDA/spack-stack) is recommended for the installation of dependent libraries.
 
-Additionally, the following tier-2 machine is supported:
-* SSEC at Univ. of Wisconsin - S4 (Note that S2S+ experiments are not fully supported)
+## Supported platforms
 
-Documentation (in progress) is available [here](https://global-workflow.readthedocs.io/en/latest/).
+The Global Workflow currently supports the following machines at the indicated tier.
+
+| HPC                                     | Tier | Notes                                                                      |
+| --------------------------------------- |:----:|:--------------------------------------------------------------------------:|
+| WCOSS2<br>NCO                           | 1    | GEFS testing is not regularly performed.                                   |
+| Hera<br>NOAA RDHPCS                     | 1    |                                                                            |
+| Hercules<br>MSU                         | 1    | Currently does not support the TC Tracker.                                 |
```

Comment:
@DavidHuber-NOAA  This table is great! I learned for example, that TC tracker is not supported.  I tried to look for an issue for this, but couldn't find one.  Any chance you could point me towards additional information.

Reply:
Sure, here is the GW issue: https://github.com/NOAA-EMC/global-workflow/issues/2639.

And here is the TC Tracker issue: https://github.com/NOAA-EMC/TC_tracker/issues/3.

Thinking about it now, I suspect we could use the Orion executables on Hercules until this is resolved.  That may be worth a test.